includes/templates/reader-activation-emails/delete-account.php

```diff
@@ -61,7 +61,7 @@
 	$social_links .
 
 	'<!-- wp:paragraph {"fontSize":"small"} -->
-	<p class="has-small-font-size">' . sprintf( /* Translators: 1: site title 2: site base address. */ __( '%1$s - %2$s', 'newspack-plugin' ), '<strong>*SITE_TITLE*</strong>', '*SITE_ADDRESS*' ) . '<br>' . sprintf( /* Translators: 1: link to site url. */ __( 'You received this email because you requested to delete your %s account.', 'newspack-plugin' ), '<a href="*SITE_URL*">*SITE_URL*</a>' ) . '</p>
+	<p class="has-small-font-size">' . sprintf( /* Translators: 1: site title 2: site base address. */ __( '%1$s — %2$s', 'newspack-plugin' ), '<strong>*SITE_TITLE*</strong>', '*SITE_ADDRESS*' ) . '<br>' . sprintf( /* Translators: 1: link to site url. */ __( 'You received this email because you requested to delete your %s account.', 'newspack-plugin' ), '<a href="*SITE_URL*">*SITE_URL*</a>' ) . '</p>
```

Comment:
This section is still using the separate `*SITE_TITLE*` and `*SITE_ADDRESS*` placeholders instead of the combined `*SITE_CONTACT*` placeholder, so if you don't have an address, the cruft can still appear:

![CleanShot 2024-08-14 at 17 18 35](https://github.com/user-attachments/assets/4ff3db87-c986-40c0-8a60-c5093107a269)

Reply:
Fixed in https://github.com/Automattic/newspack-plugin/pull/3335/commits/16f31d0ae4a46f92b36c9d338773d751a0aea180